flink-streaming-java/src/main/java/org/apache/flink/streaming/api/windowing/triggers/Trigger.java

```diff
@@ -169,7 +170,17 @@ public void clear(W window, TriggerContext ctx) throws Exception {}
 		 *                                       function (function is not part os a KeyedStream).
 		 */
 		<S extends State> S getPartitionedState(StateDescriptor<S, ?> stateDescriptor);
-	
+
+		/**
+		 *
+		 * Get an existing accumulator by <code>name</code> if it exists, Otherwise add <code>defaultAccumulator</code>.
+		 *
+		 * @param name Name of the accumulator
+		 * @param defaultAccumulator Accumulator to add, if no accumulator of the given name exists
+		 * @return The accumulator registered by the given <code>name</code> or the given default <code>defaultAccumulator</code>
+		 */
+		public <V, A extends Serializable> Accumulator<V, A> getAccumulator(String name, org.apache.flink.api.common.accumulators.Accumulator<V, A> defaultAccumulator);
```

Comment:
`org.apache.flink.api.common.accumulators.Accumulator` can be shortened to `Accumulator`